adopters/README.md

```diff
@@ -1,28 +1,3 @@
-# Karmada Adopters
-
-So you and your organisation are using Karmada? That's great. We would love to hear from you! 💖
-
-## Adding yourself
-
-[Here](../docs/casestudies/adopters.md) lists the organisations who adopted the Karmada project in production.
-
-You just need to add an entry for your company and upon merging it will automatically be added to our website.
-
-To add your organisation follow these steps:
-
-1. Fork the [karmada-io/website](https://github.com/karmada-io/website) repository.
-2. Clone it locally with `git clone https://github.com/<YOUR-GH-USERNAME>/website.git`.
-3. (Optional) Add the logo of your organisation to `static/img/supporters`. Good practice is for the logo to be called e.g. `<company>.png`.
-   This will not be used for commercial purposes. You can refer to https://github.com/karmada-io/website/pull/55.
-4. Edit the list of adopters in [adopters.md](../docs/casestudies/adopters.md).
-   You can refer to the following sample table for the format.
-
-
-   | Organization/Company Name | Website                  | Usage Scenario                            | CaseStudy                            |
-   | --------------------------- | -------------------------- | ------------------------------------------- | -------------------------------------- |
-   | My Company                | https://mycompanywebsite | Use Karmada to build multi-cloud platform | (optional) Production practice cases |
-5. Save the file, then do `git add -A` and commit using `git commit -s -m "Add MY-ORG to adopters"`.
-6. Push the commit with `git push origin main`.
-7. Open a Pull Request to [karmada-io/website](https://github.com/karmada-io/website) and a preview build will turn up.
-
-Thanks a lot for being part of our community - we very much appreciate it!
+| Organization/Company Name | Website                  | Usage Scenario                                                             | CaseStudy |
+|---------------------------| -------------------------- |----------------------------------------------------------------------------|-----------|
+| Netease                   | https://www.netease.com/ | Use Karmada to build high availability and distributed middleware platform | TBD       |
```

Comment:
New adopter items are supposed to be added at https://github.com/karmada-io/website/blob/main/src/pages/adopters.md